module-2/src/components/Home/Home.js

```diff
@@ -22,13 +22,18 @@ const Home = ({onStart}) => {
           <Text style={styles.text}>
             Parabéns por você fazer parte do programa #AceleraDev sobre React
             Native, estamos muito felizes de ter você por aqui.
-            {'\n\n'}
-            Aqui você vai encontrar os próximos passos que você precisa dar para
-            estar tudo certo para iniciar as aulas do programa. Bora lá?!
           </Text>
-          <Button primary={true} onPress={onStart}>
-            VAMOS LÁ
-          </Button>
+          {renderStart ? (
+            <>
+              <Text style={styles.text}>
+                Aqui você vai encontrar os próximos passos que você precisa dar para
+                estar tudo certo para iniciar as aulas do programa. Bora lá?!
+              </Text>
+              <Button primary={true} onPress={onStart}>
+                VAMOS LÁ
+              </Button>
+            </>
+          ) : void 0}
```

Comment:
boa! aqui você poderia fazer assim também:

```suggestion
          {renderStart && (
            <>
              <Text style={styles.text}>
                Aqui você vai encontrar os próximos passos que você precisa dar para
                estar tudo certo para iniciar as aulas do programa. Bora lá?!
              </Text>
              <Button primary={true} onPress={onStart}>
                VAMOS LÁ
              </Button>
            </>
          )}
```